pkg/chartutil/create.go

```diff
@@ -59,6 +61,18 @@ service:
   type: ClusterIP
   externalPort: 80
   internalPort: 80
+ingress:
+  enabled: false
+  # Used to create Ingress record (should used with service.type: ClusterIP).
+  # hostname: chart-example.local
```

Comment:
@thomastaylor312 It's interesting, there seems to be a lack of standards around this. For example, this comment in the charts repo asked me to do the exact opposite (comment out the `hostname` value by default, because Ingress can be enabled without a hostname): https://github.com/kubernetes/charts/pull/813